### DISCUSSION TITLE
PyVista in Scipy2021 tutorial

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#1513] [Show and tell]
I am participating in Scipy2021 conference tutorial. And I found that pyvista is used in tutorial.
![E6HJuaMVkAAVzAz](https://user-images.githubusercontent.com/7513610/125330306-8b29f080-e381-11eb-95d8-61b56c8e7517.jpeg)
We can see this in
https://github.com/jupyter-widgets/tutorial

**Comment by akaszynski** [comment]
That's awesome!  Thanks for pointing that out.

**Comment by tkoyama010** [comment]
Also [this table](https://pyviz.org/tools.html#scivis) was introduced.

**Reply by akaszynski** [reply]
Looks like `pyvista` is actually quite popular!  Makes me feel better about late night commits and weekend support PRs.

**Comment by bjlittle** [comment]
It does seem that `pyvista` and `vtk` are gaining traction and the attention of the community 🚀 

At SciPy2021, apart from @tkoyama010's lightning talk (awesome, thanks!) `pyvista` and/or `vtk` was mentioned or I found usage/reference to it from:

- https://panel.holoviz.org/reference/panes/VTK.html
  - which is quite a big deal, as it opens up usage to the `HoloViz` workflow
- https://ipygany.readthedocs.io/en/latest/
  - which you guys know all about
- https://dash.plotly.com/vtk
  - this was new to me, but again, embedding `vtk` (I think it uses `vtk.js`) in dashboards for the `plotly` community

**Comment by bjlittle** [comment]
@banesullivan It would be great if the `pyvista` core devs planned to make more of an impact at SciPy2022.

Perhaps doing a tutorial, which is a lot of work, and it's hard to secure a spot as it's highly competitive... but most certainly at least a talk in the visualisation track would be great, backed up with some lightning talks.

Between now and then I'm keen to double-down and crank the handle on https://github.com/bjlittle/geovista, mature it to a point where I migrate it into https://github.com/SciTools ownership... but aim to present a talk at SciPy2022.

What do you guys think? 🤔

**Reply by akaszynski** [reply]
I'd like for use to be able to present at SciPy2022.  I've been out of the loop for 2021 and would like to be engaged in the next one.  Hopefully we can get a slot.  @tkoyama010, I'll work with you on submitting a proposal.

**Reply by tkoyama010** [reply]
It's a great idea. We have 3 ways to approch SciPy2022.
1. [Tutorials](https://www.scipy2021.scipy.org/tutorials) - We have to make 4 hours tutorial. Like @bjlittle says it is a hard to secure a spot. But it is good to make tutorial as pyvista project..
2. [Talk & Poster Presentations](https://www.scipy2021.scipy.org/talk-poster-presentations) - This is what I did in Scipy2021. I made  poster and LT. LT is the easiest way to do presentation.
3. [Sprints](https://www.scipy2021.scipy.org/sprints) - The purpose of Sprint is support the new contributers who want to join PyVista project. Almost all the project can join this. We have to gather [good-first-issue](https://github.com/pyvista/pyvista/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) for it. It is also a good chance to discuss what we will do in this project in  face-to-face.

**Reply by akaszynski** [reply]
Let's go with 1 & 3.  I can generate the tutorials as part of our API (this needs to be in the user-guide section), and we can create a discussion/project with good first issues.

**Reply by banesullivan** [reply]
I'd love to be more involved and contribute some resources for folks to learn about PyVista during SciPy2022! I'm stretched thin right now but would totally be on board to help for next year's conference if we can get a head start on it

**Reply by tkoyama010** [reply]
Thanks for your passions. I made [project for Scipy2022](https://github.com/pyvista/pyvista/projects/3). Please work on it.

**Reply by akaszynski** [reply]
We should note that they're using `ipygany` within the example because these guys like using jupyter widgets.  I'm working on better jupyterlab support in https://github.com/pyvista/pyvista/pull/1557, and this should make for some great demos.

**Reply by adamgranthendry** [reply]
@akaszynski @banesullivan I know I'm a newbie, but I'd love to help as well! Perhaps I could assist with some of the sprints and help filter a few of the more common beginner questions and/or Windows headaches?

**Reply by akaszynski** [reply]
@adamgranthendry, any help would be greatly appreciated! Especially documentation or example PRs.

**Comment by akaszynski** [comment]
FYI, great tutorial here:
https://github.com/dmatrix/ray-core-tutorial/

We can model ours after this.
